airbyte-integrations/connectors/source-bing-ads/acceptance-test-config.yml

```diff
@@ -34,6 +34,8 @@ acceptance_tests:
             bypass_reason: "Hourly reports are disabled, because sync is too long"
           - name: geographic_performance_report_hourly
             bypass_reason: "Hourly reports are disabled, because sync is too long"
+          - name: age_gender_audience_report_hourly
+            bypass_reason: "Hourly reports are disabled, because sync is too long"
```

Comment:
Can you mention how long is it? We can tweak the `timeout_seconds` value.

Reply:
description updated; hourly data fetched is limited to 180 days (or even maybe 14 days: found different numbers on 3rd-party sites)